### PR TITLE
dropdown overflow with ellipsis

### PR DESCRIPTION
**Pull request by hwigmore**
inspired by this in Spacer Pro. i was gonna make a local override but figured it'd be useful elsewhere. 

screen shot 2018-09-14 at 12 14 08 pm

added an example to the docs. includes `title` attribute on hover
screen shot 2018-09-14 at 2 38 50 pm

@emma-floored @MagdalenaHenke would you mind taking a look?

### CODE REVIEW

**Comment by emma-floored** [comment]
oops! I actually thought we already had this

File: blocks.css [hunk 1]
```diff
@@ -909,6 +909,8 @@ a.blx-button {
   flex-direction: column;
   min-width: 150px;
   max-width: 300px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (emma-floored) [inline blocks.css]: generally when I do this, I also add `white-space: nowrap;` but I dunno if it's actually necessary

Comment (hwigmore) [reply blocks.css]: oh yeah i usually do too but for some reason forgot it here!

**Comment by hwigmore** [comment]
ok to merge?

**Comment by emma-floored** [comment]
👍

**Pull request merged**